src/status_im2/subs/communities.cljs

```diff
@@ -77,13 +77,19 @@
  (fn [[communities-enabled? search-filter communities]]
    (filterv
     (fn [{:keys [name featured id]}]
-      (and (or featured (= name "Status"))                  ;; TO DO: remove once featured communities exist
+      (and (or featured (= name "Status")) ;; TO DO: remove once featured communities exist
```

Comment:
will revert this change, but really I would prefer to nuke this Status community from subs 👿